rca/project_styleguide/templates/patterns/pages/programmes/programme_index.html

```diff
@@ -47,11 +52,28 @@ <h3>
             {% else %}
                 {# no items on any page #}
             {% endif %}
+            {% endcomment %}
+
+            <div data-mount-programmes></div>
+
+            <section class="section section--end bg bg--light">
+                <div class="section__content">
```

Comment:
This is just a stub so the header can be shown with relevant white space below – the rest of this will be implemented as part of https://projects.torchbox.com/projects/rca-website-rebuild/tickets/248 and beyond.